conf/broker.conf

```diff
@@ -68,13 +68,13 @@ zooKeeperOperationTimeoutSeconds=30
 brokerShutdownTimeoutMs=60000
 
 # Enable backlog quota check. Enforces action on topic when the quota is reached
-backlogQuotaCheckEnabled=true
+backlogQuotaCheckEnabled=false
```

Comment:
This will also disable custom quota checks on namespaces. I think we should have a way to set `backlogQuotaDefaultLimitGB` to -1 for disabling the quota check by default. That would require some small adjustment in the code check.